include/verilated_types.h

```diff
@@ -613,6 +613,14 @@ class VlQueue final {
         m_deque.insert(m_deque.begin() + index, value);
     }
 
+    // inside (set membership operator)
+    bool inside(const T_Value& value) const {
+        for (const auto& val : m_deque) {
+            if (val == value) return true;
+        }
```

Comment:
It's probably faster or at least more readable to use `std::find(...) != m_deque.end()`

Likewise below.